SEP/SEP-0003/sep-0003.md

```diff
@@ -0,0 +1,44 @@
+# Property paths with a min/max hop
+
+* Short Name: min-max-propery-path
+* SEP Number: [SEP-0003](sep-0003.md)
+* Author: [Jerven Bolleman](https://github.com/jervenbolleman)
+
+## Abstract
+
+In the SPARQL 1.1. working group, a form of bounded property path where supported in the late [drafts](https://www.w3.org/TR/2012/WD-sparql11-query-20120105/#propertypaths). The form being a path element _elt_, followed by a curly bracket enclosed min/max pattern.
+
+||Syntax Form||Matches||
+| _elt_{n,m} | A path of between n and m occurrences of elt. |
```

Comment:
I assume `m` is inclusive here?
An option would be to use the `[n,m]` notation in the description in this table (also for the next forms).

Reply:
The july 2012 spec, did not say but it is something that should be written down. For consistency with other functions, I think it should be inclusive.